app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  has_many :comments, class_name: 'comment', foreign_key: 'author_id'
+  has_many :posts, class_name: 'post', foreign_key: 'author_id'
+  has_many :likes, class_name: 'like', foreign_key: 'author_id'
+  belongs_to :author, class_name: 'User'
```

Comment:
- [ ] Great job adding the class_name property.  However, you also added more associations than needed.  This one just needs to be:

```ruby
belongs_to :author, class_name: 'User'
has_many :comments
has_many :likes
```

That way, it will be consistent with the ERD.